packages/http-client-python/generator/pygen/codegen/models/request_builder_parameter.py

```diff
@@ -112,4 +112,6 @@ def name_in_high_level_operation(self) -> str:
             return f"_{self.client_name}"
         if self.implementation == "Client":
             return f"self._config.{self.client_name}"
+        if self.is_continuation_token:
+            return f"{self.client_name} if _next_token is _Unset else _next_token"
```

Comment:
if we do not need `_Unset` as i mentioned in the generated code change pr, then we do not need this `is_continuation_token`, right?

Reply:
We still need is_continuation_token but we shall replace `_Unset` with `None`